wiki/Options/Offset_Wizard/en.md

```diff
@@ -1,41 +1,21 @@
 ---
 needs_cleanup: true
 ---
-<!-- TODO: this is basically a guide already, but there's another guide for the same thing. -->
 
 # Offset wizard
 
-If, in every single beatmap you play, you hear that the sound effects are out of sync with the song, you may want to adjust your universal offset.
+*See also: [How To Use The Offset Wizard](/wiki/Guides/How_to_Use_the_Offset_Wizard) and [universal offset](/wiki/Universal_offset)*
 
-The universal offset (also known as "offset wizard" or "offset adjustment") adjusts the audio offset so that the hitsound effects would be timed correctly.
-You can find this option in the [Options](..).
+To access the Offset Wizard, open the [Options](..) menu and type `offset` into the quick-search. Then click on the `Offset wizard` button.
 
-You can use the slider given to adjust the universal offset, however, this requires you to know what the timing needs to be.
-Instead, you can use `Offset Wizard` for a more graphical representation of ticks.
+The Offset Wizard is used to assist the user in calibrating the [universal offset](/wiki/Universal_offset) to the correct value for their device. This is necessary when the [hitsounds](/wiki/Beatmapping/Hitsound) feel out of sync with the music, across every beatmap you play. For more information on how to do this, see [How to use the Offset Wizard](/wiki/Guides%2FHow_to_Use_the_Offset_Wizard).
 
-Unlike the local song offset (specific to that difficulty only), this offset affects *every* beatmap you play.
-If you received an online offset, you can manually adjust it yourself for the difficulty you are playing using `+` or `-` to increase or decrease, respectively, the beatmap's offset by 5 milliseconds.
-You can also hold `Alt` while pressing `+` or `-` will allow you to change the offset by 1 millisecond.
+Upon opening the Offset Wizard, the user is greeted with a metronome playing and vertical green bars scrolling across the screen repeatedly. The song that was last playing/selected will begin playing, however, this can be changed at the top right of the screen via the `Now Playing` menu.
```

Comment:
```suggestion
Upon opening the Offset Wizard, the user is greeted with a metronome ticking, and vertical green bars scrolling across the screen repeatedly. The song that was last selected will begin playing, although this can be changed via the jukebox at the top right corner of the screen .
```